projects/noloan/common/src/main/java/hasadna/noloan/common/DbMessages.java

```diff
@@ -0,0 +1,174 @@
+package hasadna.noloan.common;
+
+import android.util.Log;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import hasadna.noloan.protobuf.SmsProto.SmsMessage;
+
+// Simple singleton to hold smsMessages from the db, smsMessages from the inbox
```

Comment:
`smsMessages from the inbox` - where is this stored?

Reply:
forgot to change that description